api/MockAPI.ts

```diff
@@ -39,6 +40,30 @@ class MockAPI implements AppAPI {
     };
   }
 
+  offer(): Offer {
+    return {
+        changes: {},
+        decoded: {
+          amount_msat: '100msat',
+          created_at: 1656566837,
+          description: "Offer by rusty's node",
+          features: '02000000024100',
+          min_final_cltv_expiry: 18,
+          node_id: '4b9a1fa8e006f1e3937f65f66c408e6da8e1ca728ea43222a7381df1cc449605',
+          offer_id: '28522b52ac39fa518ce3a5b3e4a9a96372487e78ba5eb1540ec4d9f02ca82718',
+          payer_info: '98b6d8e08b9b3ab7a82408b59dd9238e',
+          payer_key: '7e1714e279c4c4d05fd3c1f20bbe9b6d5cf6d47f8e9f7054fc5f476391c6c78f',
+          payment_hash: '16084e0e47f38e77d43cfce6fa6a5881aa7ffb07060c5684406502092c76ab9b',
+          relative_expiry: 7200,
+          signature: '017da6e672bbff9166d6bd15eea692e46b7cbee7b4814ab11d2ac0a8b948a6f94336e8c29d066b98ef2eab3e1f98a3d696deea4b0ae97e76cddd22e0cbf3e4fb',
+          timestamp: 1656566837,
+          type: 'bolt12 invoice',
+          valid: true
+        },
```

Comment:
Mh, this is the bolt12 invoice? I think this is the result of the fect invoice.


The offer type should be smaller like this one

`lno1pg257enxv4ezqcneype82um50ynhxgrwdajx283qfwdpl28qqmc78ymlvhmxcsywdk5wrjnj36jryg488qwlrnzyjczlqs85ck65ycmkdk92smwt9zuewdzfe7v4aavvaz5kgv9mkk63v3s0ge0f099kssh3yc95qztx504hu92hnx8ctzhtt08pgk0texz0509tk`

Take a look to the docs https://lightning.readthedocs.io/lightning-offer.7.html

The offer command has also some input parameter